shopfloor_base/models/shopfloor_app.py

```diff
@@ -109,9 +89,9 @@ def _compute_url(self):
     @api.depends("tech_name")
     def _compute_registered_routes(self):
         for rec in self:
-            routes = sorted(rec._registered_routes())
+            routes = sorted(rec._registered_routes(), key=lambda x: x.route)
```

Comment:
Yeah, TBH I'm still puzzled sometimes regarding the names because all in all the border is quite thin -> we are talking about "routing rules". However, given that this particular method returns `EndpointRule` objects, seems fair.